backend/src/main/java/com/empire/Region.java

```diff
@@ -396,6 +396,20 @@ public boolean isLand() {
 	public boolean isSea() {
 		return type == Type.WATER;
 	}
+
+	void plant() {
+		if (religion == Ideology.CHALICE_OF_COMPASSION) crops += population * Constants.chaliceOfCompassionPlantPerCitizen;
+	}
+
+	void harvest(Set<String> stoicNations, GoodwillProvider goodwills) {
+		if (!isLand()) return;
+		double maxHarvest = population * Constants.harvestPerCitizen;
+		double unrest = calcUnrest(goodwills);
+		if (unrest > .25 && !stoicNations.contains(getKingdom())) maxHarvest *= 1.25 - unrest;
+		maxHarvest = Math.min(crops, maxHarvest);
+		food += maxHarvest;
+		crops = population * Constants.plantsPerCitizen;
```

Comment:
Does this belong in the plant() method above?

Reply:
Moved it there.